seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -542,24 +542,17 @@
       "description": [
         "A new feature of ES6 or ES2015, is that it allows you to use string interpolation with back-ticks.",
         "Consider the code below",
-        "<code>const text = 'Hello World';</code>",
-        "<code>// string interpolation</code>",
-        "<code>const divText = `</code>",
-        "<code><div></code>",
-        "<code>   ${text}</code>",
-        "<code></div></code>",
-        "<code>console.log(divText); // prints </code>",
-        "<code>//<div></code>",
-        "<code>// Hello World</code>",
-        "<code>//</div></code>",
+        "<blockquote>const text = 'Hello World';<br>// string interpolation<br>const divText = `<br>&lt;div&gt;<br>   ${text}<br>&lt;/div&gt;<br>`;<br><br>console.log(divText);</blockquote>",
+        "It prints:",
+        "<blockquote>&lt;div&gt;<br>   Hello World<br>&lt;/div&gt;<br></blockquote>",
         "A lot of thing happened in there.",
-        "First, the ${variable} syntax. It's a template literal. Basically, you won't have to use concatenation with + anymore. Just drop the variable in your string, wrapped with ${ and }.",
+        "First, the <code>${variable}</code> syntax. It's a template literal. Basically, you won't have to use concatenation with + anymore. Just drop the variable in your string, wrapped with ${ and }.",
         "Second, we used backticks, not quotes, to wrap around the string. Notice that the string is multi-line.",
         "In ES6, back-ticks give you more robust string creation ability.",
         "Instructions",
         "Use proper template literal syntax with backticks.",
-        "The expected output is each entry of result object's failure array, wrapped inside an <li> element, with class attribute text-warning.",
-        "If you have trouble finding backticks, it's the ` key, to the left of your 1; and has ~ on it."
+        "The expected output is each entry of result object's failure array, wrapped inside an element, with class attribute text-warning.",
```

Comment:
Thanks for your help @systimotic You're absolutely right, I'll fix it soon and insert your superuser link as well.